test/Microsoft.TestPlatform.Utilities.UnitTests/Microsoft.TestPlatform.Utilities.UnitTests.csproj

```diff
@@ -17,9 +17,6 @@
     <ProjectReference Include="..\..\src\Microsoft.TestPlatform.Utilities\Microsoft.TestPlatform.Utilities.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeCoverage">
```

Comment:
I am blind, what is this duplicate of?

Reply:
You are not blind, I poorly tried to explain in the description that we already import this for all tests through the global TP target.